subgraphs/orbit/src/mappings/handlers.ts

```diff
@@ -0,0 +1,343 @@
+import {
+  Address,
+  BigDecimal,
+  BigInt,
+  Bytes,
+  dataSource,
+  log,
+  ethereum,
+} from "@graphprotocol/graph-ts";
+
+import { BridgeConfig } from "../sdk/protocols/bridge/config";
+import { NetworkConfigs } from "../../configurations/configure";
+
+import {
+  BridgePermissionType,
+  BridgePoolType,
+  CrosschainTokenType,
+} from "../sdk/protocols/bridge/enums";
+
+import { getUsdPrice, getUsdPricePerToken } from "../prices";
+import { TokenPricer } from "../sdk/protocols/config";
+import { Token } from "../../generated/schema";
+import { SDK } from "../sdk/protocols/bridge";
+import { CustomEventType } from "../sdk/util/events";
+import { Pool } from "../sdk/protocols/bridge/pool";
+import { Versions } from "../versions";
+import { bigIntToBigDecimal } from "../sdk/util/numbers";
+import { TokenInitializer, TokenParams } from "../sdk/protocols/bridge/tokens";
+import { _ERC20 } from "../../generated/Vault/_ERC20";
+import { networkToChainID } from "../sdk/protocols/bridge/chainIds";
+import { ERC20NameBytes } from "../../generated/Vault/ERC20NameBytes";
+import { ERC20SymbolBytes } from "../../generated/Vault/ERC20SymbolBytes";
+import { BIGINT_ZERO, getNetworkSpecificConstant } from "../sdk/util/constants";
+import { Deposit, Withdraw } from "../../generated/Vault/Vault";
+import { Swap, SwapRequest } from "../../generated/Minter/Minter";
+
+const taxReceiver = "0xe9f3604b85c9672728eeecf689cf1f0cf7dd03f2";
```

Comment:
this should be moved to common/constants.ts

Reply:
Done